pkg/util/files.go

```diff
@@ -26,6 +27,14 @@ const (
 	MaximumNewDirectoryAttempts = 1000
 )
 
+// IOFileHandles is a struct for holding CommonOptions' In, Out, and Err I/O handles, to simplify function calls.
+type IOFileHandles struct {
```

Comment:
:+1 sweet. Not sure whether I'd put it into a dedicated io.go or here, but luckily Go does not care :-)